.github/workflows/build-image.yml

```diff
@@ -72,3 +72,65 @@ jobs:
             org.opencontainers.image.revision=${{ github.sha }}
       - name: Echo image digest
         run: echo ${{ steps.docker_build.outputs.digest }}
+  rocm-image:
```

Comment:
It will build the image in sequential. Correct?

Reply:
no, it will maximize the number of jobs run in parallel depending on the available vms